### PR TITLE
Add installation instructions for embedded SDK

### PR DESCRIPTION
**Pull request by belugum**
Instructions are taken from the old documentation: https://web.archive.org/web/20161231233515/https://parse.com/apps/quickstart#embedded/raspberrypi

### CODE REVIEW

**Review by flovilmart — APPROVED** [review]
Wonderful! Did you get those tested beforehand if still working?

File: _includes/embedded_c/getting-started.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Next, navigate to the newly downloaded directory and run a few commands to build
 
 ```bash
 cd parse-embedded-sdks
+touch README
```
Comment (flovilmart) [inline _includes/embedded_c/getting-started.md]: Uhm?

**Comment by belugum** [comment]
Yes, it still works.
At first, I got an error with autoreconf:
```configure.ac:7: installing 'config/missing'
Makefile.am: error: required file './README' not found
common/Makefile.am: installing 'config/depcomp'
autoreconf: automake failed with exit status: 1
```
The workaround is creating an empty README file. Not sure why it is needed though...

**Comment by flovilmart** [comment]
Interesting! Let’s keep the workaround and figure it out later!

**Comment by flovilmart** [comment]
Thanks for the PR and taking the time to try it out :)

**Pull request merged**